docs/UserGuide.md

```diff
@@ -145,6 +145,22 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding an order under a Person : `addOrder`
+
+Adds an order under a specified person from the address book.
+
+Format: `addOrder INDEX i/ITEM d/DATE [c/QUANTITY]`
+
+* List all orders of the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index and quantity **must be a positive integer** 1, 2, 3, …​
+* Date must follow the following format: `DD-MM-YYYY`
+* If the quantity is not provided, the quantity will be set to a default value of **1**.
+
+Examples:
+* `addOrder 2 i/Lamp d/20-11-2024 c/3` 
```

Comment:
Following the deletePerson example, maybe u can say list/find followed by addOrder because we are adding by index and I don't think we can add an order without looking at list?
since there will be one for find one for list, maybe put one with count one without?
![image](https://github.com/user-attachments/assets/c377517a-dc4d-4a90-9f34-b1908ae05f1d)

Reply:
the `edit` command (for editing a person) also doesn't really mention examples of using `find` command first. Nonetheless, will add an example for this.